onnx/utils.py

```diff
@@ -133,7 +133,8 @@ def extract_model(
         input_path,  # type: Text
         output_path,  # type: Text
         input_names,  # type: List[Text]
-        output_names  # type: List[Text]
+        output_names,  # type: List[Text]
+        check_model=False,  # type: Boolean
```

Comment:
make this true by default so that existing scenarios are unaffected.

Reply:
Fixed. Thanks!